docs/Wizards_Content.md

```diff
@@ -0,0 +1,36 @@
+# Wizard Content Management Guide
+
+Wizard Management is currently under development.
+
+## How to access the management interface
+
+The Wizard Manager interface can be accessed by using the USAGov tools shortcut on the Drupal Menu Bar, and selecting Wizard Manager.
+
+## Approach
+
+The Wizard Manager allows you to manage wizards in a hierarchical tree view. Wizards are comprised of taxonomy terms that are related to each other in a parent-child relationship. 
+
+## How to create a wizard
+
+In order to create a new wizard, go to the Wizard Manager and press the button "Add term". This step will be used repeatedly in order to create a new wizard and new sub-terms and options of that header wizard. 
+
+On the Add term page, you will be asked and need to fill out the following information:
+
+* Name: Drupal's name for the taxonomy term
+* Language: Language
+* Option Name: This is the name that will show up when you are using the hierarchical tree-view within the Wizard Manager interface
+* Description: This field is for additional text that you want displayed below the option in the option list. 
+* Heading/Question: This is the heading that will display at the top of your wizard question, in large red text. 
+* Language Toggle: Sets a translation for the current page and the translation goes into the language dropdown in the top right
```

Comment:
should be "translation is linked to in the language switch link in the top right"